controllers/actions-controller.js

```diff
@@ -0,0 +1,93 @@
+const db = require('../models');
+
+module.exports = (app) => {
+
+  // Show action
+  app.get('/actions/:id', (req, res) => {
+    const actionId = req.params.id
+    db.Action.findById(actionId)
+    .then((action) => {
+      console.log("Response from action/Show: ", action)
+      res.status(200)
+      res.json({
+        message: "action request successful",
+        action
+      })
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(400);
+      res.json({
+        message: "Error!",
+        error: err
+      })
+    })
+  });
+
+  //TODO: Code Review  whether the actionId exists, it does not ever hits catch error
+    // UPDATE a action
+  app.put('/actions/:id/update', (req, res) => {
+    const actionId = req.params.id
+    const action = req.body
+    db.Action.update(action, {
+      where: { id: actionId }
+    }).then((response) => {
+        res.status(200)
+        res.json({
+          msg: 'action updated successfully!',
+        })
+      }).catch((err) => {
+        console.log(err);
+        res.status(400);
+        res.json({
+          message: "Error!",
+          error: err
+        })
+      })
+    });
+
+  // Create a action
+  app.post('/criteria/:criterionId/actions/create', (req, res) => {
```

Comment:
I would highly recommend to be consistent with naming things... it's easy to get this very confusing. In some places it's criteria and in other it's criterion. I'd pick one and stick with it.

Reply:
Isn't the RESTful convention to name resources plural unless they are referring to model?  Criterion is the singular of Criteria.

Reply:
I agree with @tlambrou , the Model is called Criterion. I think we can accept the pull request